composer.json

```diff
@@ -32,11 +34,19 @@
     "config": {
         "bin-dir": "bin"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/diablomedia/phpstorm-stubs"
```

Comment:
I'm using a fork of the stubs because it makes some changes that makes the stubs usable when used inside a PHP runtime (that's not their intended use as-is).  I've also fixed many return type-hints that weren't documented quite right (I'll try to make PRs against the jetbrains repo for most of these).